packages/common/src/MultiVersionTestHelpers.js

```diff
@@ -0,0 +1,134 @@
+const web3 = require("web3");
```

Comment:
This Util pulls out the common functions and constants used in the previous PR to generalize the multi-version testing within the repo. These can be re-used through protocol.